### DISCUSSION TITLE
[Bug]: Plugins Menu Items not shown

### DISCUSSION THREAD

**Discussion by robertkowalski** [danny-avila/LibreChat#2067] [Troubleshooting]
### What happened?

I try to install a plugin for librechat, however, i can't access the store

### Steps to Reproduce

1. setup librechat with open ai azure
2. try to select "plugins"

### What browsers are you seeing the problem on?

Chrome

### Screenshots

<img width="642" alt="Screenshot 2024-03-11 at 15 51 27" src="https://github.com/danny-avila/LibreChat/assets/298166/b2a91a3e-ebce-4582-9572-0ac43beac664">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
https://docs.librechat.ai/install/configuration/azure_openai.html#using-plugins-with-azure
